tests/integration-tests/network_template_builder.py

```diff
@@ -248,6 +248,24 @@ def __build_vpc_endpoints(self, subnet_id, route_table_ids):
                 type=VPCEndpointConfig.EndpointType.INTERFACE,
                 enable_private_dns=True,
             ),
+            VPCEndpointConfig(
+                name="SystemsManagerEndpoint",
+                service_name=f"com.amazonaws.{region}.ssm",
```

Comment:
should we add them here: https://docs.aws.amazon.com/parallelcluster/latest/ug/network-configuration-v3.html#aws-parallelcluster-in-a-single-public-subnet-no-internet-v3 ?

Reply:
I don't think so, because
"To ensure the proper functioning of the cluster, the cluster nodes must be able to interact with a number of AWS Services. .. [...]"
SSM service is not a cluster requirement, but let's talk.